### PR TITLE
buyer can decided eb entity

### PR DESCRIPTION
**Pull request by sarias-eb**

### CODE REVIEW

File: supplier_app/tests/test_view.py [hunk 1]
```diff
@@ -1741,6 +1742,7 @@ def test_get_template_as_supplier_redirects_to_supplier_home(self):
     def test_valid_company_creation(self):
         self.client.force_login(self.ap_user)
         self._make_post()
+        # import ipdb; ipdb.set_trace()
```
Comment (nicolasnunez-eb) [inline supplier_app/tests/test_view.py:1745]: Remove this please

Comment (sarias-eb) [reply supplier_app/tests/test_view.py]: Done :)

Comment (ricardocaif-eb) [inline supplier_app/tests/test_view.py]: NIT: remove comment of debugger

**Comment by sarias-eb** [comment]
The buyer needs to link the company and the eb entity to bill in the create company form

File: supplier_app/models.py [hunk 2]
```diff
@@ -42,9 +42,26 @@
...
+    )
+
+    def __str__(self):
+        return self.eb_name
+
+
 class Company(models.Model):
     name = models.CharField(max_length=200, verbose_name=_("Name"))
     description = models.TextField()
+    eb_entity = models.ForeignKey(EBEntity, on_delete=models.CASCADE, default=None)
```
[the remark this answers is not in the document]
Comment (sarias-eb) [reply supplier_app/models.py:64]: The company needs to know who eb entity is going to bill to show the correct information in create taxpayer

**Pull request merged**